src/helperFunctions/config_deprecated.py

```diff
@@ -31,32 +31,6 @@ def get_config_dir():
     return f'{get_src_dir()}/config'
 
 
-def read_list_from_config(config_file: ConfigParser, section: str, key: str, default=None):
-    '''
-    Parses a comma separated list in section `section` with key `key`.
-
-    :param config_file: The FACT configuration
-    :param section: The section to read from
-    :param key: The key holding the list
-
-    :return: A list holding the values defined in the config
-    '''
-    if default is None:
-        default = []
-
-    if not config_file:
-        return default
-
-    try:
-        config_entry = config_file.get(section, key)
-    except (NoOptionError, NoSectionError):
-        return default
-
-    if not config_entry:
-        return default
-    return [item.strip() for item in config_entry.split(',') if item]
-
-
 def get_temp_dir_path(config: ConfigParser = None) -> str:
```

Comment:
To deprecated this function I'd propose to semantically analyse the config after loading.
One part of the analysis would be to ensure that the temporary directory exists.
Imo it does not make sense to automatically create it, since the sysadmin explicitly decided to differ from `/tmp` and it can be expected that he/she creates the directory.

Reply:
Done.